src/common/electron-telemetry-events.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this file be in /src/electron rather than /src/common?

Reply:
Good catch. Fixed.